core-web/libs/dotcms-scss/jsp/scss/backend/dot-admin/portlets/_view-workflow.scss

```diff
@@ -7,11 +7,6 @@
     height: 6px;
 }
 
-.portlet-sidebar .dijitCheckBox {
```

Comment:
why ? no references in the code ?

Reply:
It was used to center this checkbox, but is not needed anymore

<img width="1512" alt="Screenshot 2023-11-07 at 10 26 03 AM" src="https://github.com/dotCMS/core/assets/63567962/6899f98b-a5da-498b-bb26-ccdd01e47c93">